### DISCUSSION TITLE
Domain Name & Custom Email Address Linkage

### DISCUSSION THREAD

**Discussion by johnhuatech** [vercel/community#2459] [Help] [answered]
### Summary

I am interested in purchasing Vercel for commercial deployment. I am wondering if purchasing their domain name will come with custom email addresses and other cPanel like functions that are normally sold together as a package with the domain name, more than as-is. 

Thanks

**Comment by smaeda-ks** [comment]
Hi @johnhuatech , 

1. We do not provide any Email services. You need to use a third-party provider for email hosting.
2. We do provide DNS service (you will be able to add custom DNS records just like you would do with any normal domain registrar).
https://vercel.com/docs/concepts/projects/domains/managing-dns-records

**Comment by johnhuatech** [comment]
Can I register a domain from a third party with email onto a deployment
that I develop with vercel?

On Mon, May 15, 2023 at 8:21 PM Shohei Maeda ***@***.***>
wrote:

> Hi @johnhuatech <https://github.com/johnhuatech> ,
>
>    1. We do not provide any Email services. You need to use a third-party
>    provide for email hosting.
>    2. We do provide DNS service (you will be able to add custom DNS
>    records just like you would do with any normal domain registrar).
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/vercel/discussions/2459#discussioncomment-5911540>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A35PSBQII2XUJNECUVMWA3DXGLXFLANCNFSM6AAAAAAYCIZWFY>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>
-- 
--
--
*CONFIDENTIALITY NOTICE: This message and any accompanying documents
contain information belonging to the sender which may be confidential and
legally privileged. This information is only for the use of the individual
or entity to which it was intended. If you are not the intended recipient,
any disclosure, copying, distribution, or action taken in reliance on the
contents of the information contained in this message and any accompanying
documents is strictly prohibited. If you have received this message in
error, please contact the sender immediately and delete the message.*



*Sender reserves all rights to disclosure in the event of client-attorney
privileges, as outlined in legal situations, in limitation as not
obstructive to current standing legal public records acts. Receiving party
may be held responsible for any fees associated proceeding legal fees in
the event of wrongful disclosure. We calculate any additional arbitration
damages based on daily computational compounding interest at cpi rate.
Electronics communication receiver is notified to these terms and
conditions on response and receipt of this electronic communications.John
Hua is a business, engineering, civics professional pursuing higher
educational studies on multidisciplinary topics. These studies have
included: International relations, Education, Economics, Ethnicity
sociology, Management systems in addition to his three core subject focus.
He has received his business education from Future Business Leaders of
America as a west coast section competitive champion. He has mechatronics,
astrophysics, and software engineering education from Johns Hopkins
University, Project Lead the Way Academy, and Google. His civics education
arrives from a Civic Education Project in partnership with Northwestern
University and UC Berkeley along with the Ludwick Foundation Arsalyn
Program. The American Legion Boys State- California Sacramento State
Assembly and Senate, Los Angeles County Councils, US Department of
Education, and Jeff Sein Kwan Yang Scholars have all provided recognitional
awards in his work. APIA Scholars, one of the largest philanthropic
non-profit based in Washington DC with partnerships such as the Hispanic
Scholar Fund, Gates Millennium Program, and Coca-Cola Scholars, of a $15
Million dollar portfolio, continues to support his progress. He plans to
finish a high degree of education and work in pursuit of opportunities
including, but not limited to- Harvard Law, Stanford Graduate School of
Business, and Massachusetts Institute of Technology Sloan. *

**Comment by smaeda-ks** [comment]
@johnhuatech Sure, you can buy domains in any registrar, and use that in your Vercel projects. You just need to create the right DNS records to point the domain to Vercel, which you will be instructed upon adding domains to your project on the Vercel UI.

**Comment by johnhuatech** [comment]
Got it. It's not a redirection to .vercel but a pointer correct so the
website can show xyz.com

Is there email forwarding if I pay the $20 for a vercel dns? I assume your
team earns more that way though typically hosting sites tend to also
include email options.

On Mon, May 15, 2023 at 8:37 PM Shohei Maeda ***@***.***>
wrote:

> @johnhuatech <https://github.com/johnhuatech> Sure, you can buy domains
> in any registrar, and use that in your Vercel projects. You just need to
> create the right DNS records to point the domain to Vercel, which you will
> be instructed upon adding domains to your project on the Vercel UI.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/vercel/discussions/2459#discussioncomment-5911611>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A35PSBRWMMWVW7SVRVW7KKTXGLY7DANCNFSM6AAAAAAYCIZWFY>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>
-- 
--
--
*CONFIDENTIALITY NOTICE: This message and any accompanying documents
contain information belonging to the sender which may be confidential and
legally privileged. This information is only for the use of the individual
or entity to which it was intended. If you are not the intended recipient,
any disclosure, copying, distribution, or action taken in reliance on the
contents of the information contained in this message and any accompanying
documents is strictly prohibited. If you have received this message in
error, please contact the sender immediately and delete the message.*



*Sender reserves all rights to disclosure in the event of client-attorney
privileges, as outlined in legal situations, in limitation as not
obstructive to current standing legal public records acts. Receiving party
may be held responsible for any fees associated proceeding legal fees in
the event of wrongful disclosure. We calculate any additional arbitration
damages based on daily computational compounding interest at cpi rate.
Electronics communication receiver is notified to these terms and
conditions on response and receipt of this electronic communications.John
Hua is a business, engineering, civics professional pursuing higher
educational studies on multidisciplinary topics. These studies have
included: International relations, Education, Economics, Ethnicity
sociology, Management systems in addition to his three core subject focus.
He has received his business education from Future Business Leaders of
America as a west coast section competitive champion. He has mechatronics,
astrophysics, and software engineering education from Johns Hopkins
University, Project Lead the Way Academy, and Google. His civics education
arrives from a Civic Education Project in partnership with Northwestern
University and UC Berkeley along with the Ludwick Foundation Arsalyn
Program. The American Legion Boys State- California Sacramento State
Assembly and Senate, Los Angeles County Councils, US Department of
Education, and Jeff Sein Kwan Yang Scholars have all provided recognitional
awards in his work. APIA Scholars, one of the largest philanthropic
non-profit based in Washington DC with partnerships such as the Hispanic
Scholar Fund, Gates Millennium Program, and Coca-Cola Scholars, of a $15
Million dollar portfolio, continues to support his progress. He plans to
finish a high degree of education and work in pursuit of opportunities
including, but not limited to- Harvard Law, Stanford Graduate School of
Business, and Massachusetts Institute of Technology Sloan. *

**Comment by smaeda-ks** [comment, marked as answer]
@johnhuatech what do you mean by "email forwarding"? Could you be more specific on your questions?

If you're asking whether you can create `MX` records, then yes, you can do so. What I'm saying is, you don't have to buy domains on Vercel, it's nice to have, but completely optional.

If you decide to buy a domain with a third-party registrar (e.g. GoDaddy), that's totally fine. You can still use that domain with your Vercel projects, by either pointing the domain to Vercel via `A`/`CNAME` records, or, switching name servers to Vercel
https://vercel.com/docs/concepts/projects/domains/add-a-domain#vercel-nameservers

**Reply by bryandevs1** [reply]
I am having a problem with that now. if i buy the domain from another provider, and add it to vercel, how do i configure the email because i am no loner receiving mails to that address

**Reply by smaeda-ks** [reply]
> because i am no loner receiving mails to that address

That's not true. You're only using Vercel for your web properties, and emails will continue to work with your existing `MX` records. `A` and `MX` records don't conflict with each other and can co-exist.
https://vercel.com/docs/concepts/projects/domains/troubleshooting#working-with-apex-domain
